### DISCUSSION TITLE
How to run AMSG on Godot 3 or get access to Godot 4?

### DISCUSSION THREAD

**Discussion by jayypluss** [ywmaa/Advanced-Movement-System-Godot#3] [Q&A] [answered]
I'm trying to run AMSG with the latest Godot version I've got, which is 3.3.4-2 on Arch Linux, but I won't open, I believe it's because it was made for Godot 4? 
But how do as far as I know Godot 4 version wasn't released yet, so how do I get access to Godot 4? 
Do you compile it yourself from Godot latest source code? 
Could someone give any basic instructions to run AMSG?

**Comment by JWBWork** [comment, marked as answer]
I believe to use Godot 4 right now you have to go to Godot's github page compile the app yourself

[I think the docs should cover it here](https://docs.godotengine.org/en/stable/development/compiling/index.html)

**Reply by jayypluss** [reply]
Thanks man!

**Reply by ywmaa** [reply]
thanks for answering .

for some reason I didn't get a notification about the discussion

**Reply by ywmaa** [reply]
there is also another way to try Godot 4

those are pre-alpha testing :

https://downloads.tuxfamily.org/godotengine/testing/4.0/



if anyone tries to open the project please tell me if there is a problem, because someone already told me that it is not opening the project .
